acceptance/tests/custom_facts/using_win32ole_should_not_hang.rb

```diff
@@ -0,0 +1,29 @@
+test_name 'C92060: Custom facts should not hang Facter when using win32ole' do
+  tag 'risk:high'
+
+  confine :to, :platform => /windows/
+  require 'timeout'
+
+  content = <<EOM
+Facter.add('custom_fact') do
+  setcode do
+    require 'win32ole'
+    locator = WIN32OLE.new('WbemScripting.SWbemLocator')
+    locator.ConnectServer('', "root/CIMV2", '', '', nil, nil, nil, nil).to_s
+  end
+end
+EOM
+
+  agents.each do |agent|
+    custom_dir = agent.tmpdir('arbitrary_dir')

```

Comment:
Recommend a `teardown` to remove this directory as has been done in the other tests.